adr/0006-reviewer-responsibilies.adoc

```diff
@@ -0,0 +1,74 @@
+= Reviewer and committer responsibilities
+
+* Status: proposed
+* Date: 2024-10-25
+
+== Context and Problem Statement
+
+Today, we give different roles, responsibilities and resource access across Quarkus contributors. 
+It is not well known nor written down what the review and push responsibilities are and how they ares structured to achieve Quarkus goals of high code quality and velocity.
+
+This leads to different interpretations and some unknown on how contributors can best help.
+At the same time, some contributions remain unreviewed for a long time because of uncertainty or because of bottleneck.
+
+== Decision Drivers
+
+* Increase clarity and awareness on contributor’s responsibility on reviews and code push
+* Describe the intent and not the current exact state (as tooling is preventing us from having the best solution)
+* Being a perfect solution is _not_ a goal, _better_ than today is.
+* Define ways to measure and improve the time to review.
+
+== Scenarios (optional)
+
+List of scenarios, each described using 2-3 sentences, to capture where the ADR has an impact.
+
+== Considered options
+
+The main alternative is the status quo which roughly is: Guillaume knowing what we are intending to do and some persons talking with Guillaume regularly being sufficiently aware.
```

Comment:
That's not true. While Guillaume provides extensive review on many areas, he is not alone to overlook the whole thing.

Reply:
This sentence is not talking about the review, it is talking about the process for the reviews.